src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -229,6 +229,8 @@ def verify_workspace_client(workspace_client: WorkspaceClient) -> WorkspaceClien
         workspace_client.config._product = "remorph"
     if workspace_client.config._product_version != __version__:
         workspace_client.config._product_version = __version__
+
+    logger.info(f"user: {workspace_client.current_user.me().user_name}")
```

Comment:
can you remove that from CLI so we don't log twice.

Reply:
I have checked @sundarshankar89 
Transpile:
In cli.py, we don't call for `verify_workspace_client`, so this is logged only once.
We call `verify_workspace_client` in experimental functions, which are not related to CLI, so this shouldn't be a problem.

Reconcile:
In reconcile we log the `user` in CLI and trigger the job. 
In the workflow job, while running the execute.py/recon function, we log it in the workflow job logs, so it does not log twice.

Please let me know if you still want me to remove it from reconcile CLI.

Reply:
To follow up on your suggestion, I have removed the logger from cli.py/transpile and called `verify_workspace_client` to execute.py/morph function. This should cover the CLI and any cx wrapper around the morph function.